code/modules/mob/living/carbon/human/life/handle_mutations_and_radiation.dm

```diff
@@ -73,3 +97,201 @@
 				var/datum/organ/external/O = pick(organs)
 				if(istype(O))
 					O.add_autopsy_data("Radiation Poisoning", damage)
+	else
+		rad_tick = max(rad_tick-1,0)
+
+	if(rad_tick)
+		/*
+		A list of events that are more likely to occur based on time exposed and amount of exposure
+
+		6 'gates'
+		100
+			Vomiting - Early on symptom, if rad_tick and radiation is above a certain level
+			Nosebeeds - Early onset of the blood vessels breaking down
+		200
+			Internal hemorrhaging - Lower rad tick has lower probability, but this ramps up as it goes along
+		300
+			Organ Damage - A little later on, again lower rad tick has lower prob
+		400
+			Minor limb mutation - We're talking a decently large exposure causing this, somewhat rare
+			Blindness - About the same chance as limb mutation
+			Clone damage - To emulate DNA breakdown
+		500
+			Necrosis - After Limb mutation
+			Glowing slightly green - Extremely rare, practically terminal, glowing increases with radiation level
+		600
+			Ghoulification if extremely lucky - We're talking 1 in 1000
+
+		For anything affecting internal or external organs, if it's robotic, exempt them.
+		*/
+
+		//For symptoms that you want to scale off of a minimum radiation amount
+		var/minor_rad_multiplier = max(1, radiation/20)
+		var/rad_multiplier = max(1, radiation/50)
+		var/major_rad_multiplier = max(1, radiation/70)
+		var/extreme_rad_multiplier = max(1, radiation/100)
+
+		if(rad_tick > RADDOSELIGHT)
+			if(prob(5*rad_multiplier))
+				//Vomit
+				vomit()
+			if(prob(5*minor_rad_multiplier))
+				//Nosebleed
+				if(prob(15))
+					to_chat(src, "<span class = 'danger'>Your nose starts bleeding!</span>")
+				drip(1)
```

Comment:
Change the drip proc to return 1 on success (it's in code/modules/organs/blood.dm @ 171), then only display the message if the drip is successful

otherwise species with no blood will get nosebleed messages which is going to destroy immersions bigly